exasol_script_languages_developer_sandbox/lib/random_string_generator.py

```diff
@@ -1,8 +1,17 @@
+import base64
 import uuid
 
 
-def get_random_str(length: int) -> str:
+def get_random_str_of_length_n(length: int) -> str:
     """
     Creates a random string of given length.
     """
-    return uuid.uuid4().hex[:length]
+    return get_random_str()[:length]
+
+
+def get_random_str() -> str:
+    """
+    Creates a random string of length 22.
+    Uses uuid.uuid4() as base and replaces "+","/" with "A"/"B" and removes any filling characters ("=").
```

Comment:
ok, I probably had removed + and  /, but this way you have a constant length. Maybe, add explanation, why base64

Reply:
Done